# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1377370033 14400
#      Sat Aug 24 14:47:13 2013 -0400
# Node ID 4daf633100db8d1fc161917588ed452b2c525984
# Parent  fe6518a1d87c9053ee326f4d7cb2dde8bc4c5fba
Associate \circ with degree symbol, for compatibility (bug #39828).

* libinterp/corefcn/oct-tex-symbols.in: Associate "circ" symbol with
0x00B0 (degree character), for compatibility.

diff --git a/libinterp/corefcn/oct-tex-symbols.in b/libinterp/corefcn/oct-tex-symbols.in
--- a/libinterp/corefcn/oct-tex-symbols.in
+++ b/libinterp/corefcn/oct-tex-symbols.in
@@ -83,17 +83,17 @@ heartsuit       0x2665  0xF0A9
 spadesuit       0x2660  0xF0AA
 leftrightarrow  0x2194  0xF0AB
 leftarrow       0x2190  0xF0AC
 Leftarrow       0x21D0  0xF0DC
 uparrow         0x2191  0xF0AD
 rightarrow      0x2192  0xF0AE
 Rightarrow      0x21D2  0xF0DE
 downarrow       0x2193  0xF0AF
-circ            0x25CB  0xF0B0
+circ            0x00B0  0xF0B0
 pm              0x00B1  0xF0B1
 geq             0x2265  0xF0B3
 propto          0x221D  0xF0B5
 partial         0x2202  0xF0B6
 bullet          0x2022  0xF0B7
 div             0x00F7  0xF0B8
 neq             0x2260  0xF0B9
 aleph           0x2135  0xF0C0
